### ISSUE TITLE
Youtube video links on front page should be more accessible

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7224]
The links should turn "active" (red play icon) when receiving keyboard focus, not only on mouse-over. (Or we drop the active/inactive distinction)

If we can hide the play icon and the "state" from the accessibility tree we should, it doesn't add anything to a visually impaired user in terms of navigation. Perhaps it can be implemented in terms of css somehow...
